tests/ctl/core/test_api.py

```diff
@@ -60,19 +71,44 @@ def test_generate_resource_urls_with_id(test_config: FidesConfig) -> None:
 class TestCrud:
     @pytest.mark.parametrize("endpoint", model_list)
     def test_api_create(
-        self, test_config: FidesConfig, resources_dict: Dict, endpoint: str
+        self,
+        generate_auth_header,
+        test_config: FidesConfig,
+        resources_dict: Dict,
+        endpoint: str,
     ) -> None:
         manifest = resources_dict[endpoint]
         print(manifest.json(exclude_none=True))
+        token_scopes: List[str] = [f"{CLI_SCOPE_PREFIX_MAPPING[endpoint]}:{CREATE}"]
+        auth_header = generate_auth_header(scopes=token_scopes)
         result = _api.create(
             url=test_config.cli.server_url,
             resource_type=endpoint,
             json_resource=manifest.json(exclude_none=True),
-            headers=test_config.user.auth_header,
+            headers=auth_header,
         )
         print(result.text)
         assert result.status_code == 201
 
+    @pytest.mark.parametrize("endpoint", model_list)
+    def test_api_create_wrong_scope(
+        self,
+        generate_auth_header,
+        test_config: FidesConfig,
+        resources_dict: Dict,
+        endpoint: str,
+    ) -> None:
+        manifest = resources_dict[endpoint]
+        token_scopes: List[str] = [PRIVACY_REQUEST_CREATE]
+        auth_header = generate_auth_header(scopes=token_scopes)
+        result = _api.create(
+            url=test_config.cli.server_url,
+            resource_type=endpoint,
+            json_resource=manifest.json(exclude_none=True),
+            headers=auth_header,
+        )
+        assert result.status_code == 403
+
```

Comment:
No tests broke when I split out the cli-objects:* scopes into individual scopes, so for each crud endpoint I'm adding a test that fails if it has the wrong scope, and a test that passes with just the singular correct scope.